src/components/content/policies/EditPolicy.tsx

```diff
@@ -0,0 +1,280 @@
+                        policyContent.current = content;
+                        form.setFieldsValue({ policy: content });
+                        files.current[0].status = 'done';
+                        setRegoSyntaxValidationStatus('completed');
+                    } catch (e: unknown) {
+                        files.current[0].status = 'error';
+                        setRegoSyntaxValidationStatus('error');
+                        if (e instanceof Error) {
+                            return (
+                                <div className={'submit-alert-tip'}>
+                                    {' '}
+                                    <Alert
+                                        message={e.message}
+                                        description={
+                                            <PolicySubmitResultDetails msg={'Policy file upload failed'} uuid={''} />
+                                        }
+                                        showIcon
+                                        closable={true}
+                                        type={'error'}
+                                    />{' '}
+                                </div>
+                            );
+                        } else {
+                            return (
+                                <div className={'submit-alert-tip'}>
+                                    {' '}
+                                    <Alert
+                                        message={'unhandled error occurred'}
+                                        description={
+                                            <PolicySubmitResultDetails msg={'Policy file upload failed'} uuid={''} />
+                                        }
+                                        showIcon
+                                        closable={true}
+                                        type={'error'}
+                                    />{' '}
+                                </div>
+                            );
+                        }
+                    }
+                }
+            };
+            reader.readAsText(uploadedFiles[0] as RcFile);
+        }
+    }
+
+    const setPolicyContentFileData = (file: RcFile): boolean => {
+        files.current.pop();
+        files.current.push(file);
+        setRegoSyntaxValidationStatus('notStarted');
+        validateAndLoadPolicyContentFile([file]);
+        return false;
+    };
+
+    const onRemovePolicyContentFile = () => {
+        files.current.pop();
+        policyContent.current = '';
+        setRegoSyntaxValidationStatus('notStarted');
+        updatePoliciesManagementServiceRequest.reset();
+    };
+
+    const setValidator = (_: unknown, uploadFiles: UploadFile[]) => {
+        if (uploadFiles.length === 0) {
+            return Promise.reject(new Error('please upload policy file!'));
```

Comment:
i dont see this error message. Instead i see two different error messages. in that the second one is not needed too.

![image](https://github.com/eclipse-xpanse/xpanse-ui/assets/54129659/0a6acddf-dc82-4112-8712-e9c4be3023de)